docs/docs/building-from-source.md

```diff
@@ -5,15 +5,15 @@ Users are recommended to use the [pre-built xemu binaries](https://github.com/mb
 ## Windows
 
 Windows builds are cross-compiled from Ubuntu. If you would like to build *on* Windows, you can use WSL2 and Docker. See [official Docker
-documentation](https://docs.docker.com/docker-for-windows/wsl/) for how to get WSL2 and Docker set up.
+documentation](https://docs.docker.com/docker-for-windows/wsl/) for how to get WSL2 and Docker set up. In the instructions below for windows, It can be used in Powershell/Command prompt but is recommended to do this in your wsl2 terminal on windows.
 
 ```bash
+# Configure git NOTE: Change back To true after use if needed
```

Comment:
Why is this necessary? I haven't had to do this on my Windows Docker setup, I don't think this needs to be added

Reply:
When running under powershell without that configuration change. It doesn't build at all.

Reply:
testing on windows, without this it builds perfectly fine, are you sure that this isn't just a misnomer

Reply:
Yes Infact I couldn't build without it at all. It would just not get the docker image for building and if the image is there already in docker and working I haven't tested that part because getting it to get the image in first place wasn't working via powershell. I'm very positive it wasn't a misnomer. The only method I found that works without it is using the wsl2 terminal. When doing these commands in powershell with git for windows it fails building. Turning it off however wouldn't effect Linux since it already is using line endings it properly needs.

Reply:
This is currently what happens when both autocrlf and recurse-submodules is removed.

Current (autocrlf on and no submodules):
![Screenshot 2022-02-05 171850](https://user-images.githubusercontent.com/64176728/152664445-173de1b5-5691-477d-b743-4c44c7e47ca0.png)
d

with autocrlf off no submodules, it doesn't finish building and just ends close to the end. (don't mind the build time used an old pc that was slow. It does the same thing on every Windows pc I try.): 
![Screenshot 2022-02-05 202413](https://user-images.githubusercontent.com/64176728/152667750-12ab9522-ee89-42c9-9e3a-82b12e21c17a.png)

having recurse-submodules lets it finish to end with a successful exe.